packages/api/cms-api/src/mikro-orm/migrations/Migration20230302145445.ts

```diff
@@ -3,7 +3,15 @@ import { Migration } from "@mikro-orm/migrations";
 export class Migration20230302145445 extends Migration {
     async up(): Promise<void> {
         this.addSql('alter table "PageTreeNode" add column "updatedAt" timestamp with time zone;');
-        this.addSql('update "PageTreeNode" SET "updatedAt"=CURRENT_DATE');
+
+        const rows = (await this.execute('select id from "PageTreeNode"')) as Array<{ id: string }>;
+
+        for (const row of rows) {
+            this.addSql(
+                `update "PageTreeNode" SET "updatedAt" = (select "Page"."updatedAt" from "PageTreeNode" left join "PageTreeNodeDocument" on "PageTreeNode"."id" = "PageTreeNodeDocument"."pageTreeNodeId" left join "Page" on "PageTreeNodeDocument"."documentId" = "Page"."id" where "PageTreeNode"."id" = '${row.id}') where id = '${row.id}'`,
```

Comment:
Page table is defined in application, you can't reference it in the library.

imho it was good enough with current date. If an application want's this correct, it has to create it's own migration. (if we consider this very important, we could create such a migration in comet-upgrade)

Reply:
I made the proposal to set the updatedAt timestamp based on the attached document, but I haven't thought it through completely. Of course this cannot work in the library...